### PR TITLE
[bitnami/minio] allow additional from clauses in NetworkPolicy

### PR DESCRIPTION
**Pull request by irizzant**
**Description of the change**
Currently if `networkPolicy.allowExternal` is `false` the `podSelector` added in the `from` clause allows connection from Pods having pre-defined labels, which is not always applicable/desiderable.

This PR adds the possibility for for additional `from` clauses in `NetworkPolicy` to add custom behavior.

**Benefits**

This PR adds the possibility for for additional `from` clauses in `NetworkPolicy` to add custom behavior.

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/minio/values.yaml [hunk 1]
```diff
@@ -746,6 +746,14 @@ networkPolicy:
   ## listening on. When true, MinIO&reg; will accept connections from any source (with the correct destination port).
   ##
   allowExternal: true
+
```
Comment (rafariossaa) [inline bitnami/minio/values.yaml:749]: Hi, 
  Could you take a look to the linter message ?

**Review by rafariossaa — CHANGES_REQUESTED** [review]

File: bitnami/minio/values.yaml (hunk 1 above)
Comment (irizzant) [reply bitnami/minio/values.yaml:749]: Hi
I don't know how to fix
`Error: [trailing-spaces] trailing spaces`

How do I run the lint?

Comment (irizzant) [reply bitnami/minio/values.yaml]: Ok found the error, I pushed the fix and the checks are now passing

**Comment by rafariossaa** [comment]
Thanks for spotting this out, and sending a PR to fix it.

**Review by rafariossaa — APPROVED** [review]
LGTM

**Comment by irizzant** [comment]
You're most welcome @rafariossaa , I also opened https://github.com/bitnami/charts/pull/8151 , I forgot to update the `README.md` sorry for that

**Comment by rafariossaa** [comment]
Hi,
  Don't worry,  it will be reviewed and merged.

**Pull request merged**